lib/wasi/src/state/mod.rs

```diff
@@ -128,10 +128,18 @@ pub struct Fd {
 }
 
 impl Fd {
+    /// This FD can be used with read system calls.
```

Comment:
FD is unexplained, you could either write "file descriptor" here, or "descriptor" or document "pub struct Fd" above?

Reply:
Yeah that should be ```[`Fd`]```, good call